apps/src/templates/instructions/codeReview/Comment.jsx

```diff
@@ -72,6 +70,57 @@ class Comment extends Component {
     return <div style={styles.error}>{javalabMsg.commentUpdateError()}</div>;
   };
 
+  toggleHideResolved = () => {
+    this.setState({hideWhileResolved: !this.state.hideWhileResolved});
```

Comment:
same comment about using state from the react updater callback rather than referencing `this.state` directly to ensure idempotency

Reply:
Thanks!